perfscale_regression_ci/scripts/cluster_density_env.sh

```diff
@@ -0,0 +1,8 @@
+# Where to clone e2e-benchmarking from
+export E2E_BENCHMARKING_REPOSITORY=${E2E_BENCHMARKING_REPOSITORY:-"https://github.com/cloud-bulldozer/e2e-benchmarking"}
+export E2E_BENCHMARKING_BRANCH=${E2E_BENCHMARKING_BRANCH:-"master"}
+# Common ENVs for kube-burner 
+export WORKLOAD=${WORKLOAD:-"cluster-density"}
```

Comment:
let's just set this WORKLOAD parameter in the tests env.sh file. If we set a different workload than custom it won't get overwritten